sql/cache.sql

```diff
@@ -11,6 +11,5 @@ CREATE TABLE IF NOT EXISTS  _timescaledb_cache.cache_inval_hypertable();
 -- etc.
 CREATE TABLE IF NOT EXISTS  _timescaledb_cache.cache_inval_extension();
 
--- not actually strictly needed but good for sanity as all tables should be dumped.
-SELECT pg_catalog.pg_extension_config_dump('_timescaledb_cache.cache_inval_hypertable', '');
-SELECT pg_catalog.pg_extension_config_dump('_timescaledb_cache.cache_inval_extension', '');
+GRANT SELECT ON ALL TABLES IN SCHEMA _timescaledb_cache TO PUBLIC;
```

Comment:
I don't think we need this now, right?

Reply:
Unfortunately we do cause pg_dump is still trying to dump those tables.

Reply:
OK pg_dump is not actually dumping those anymore without this but it will still fail cause when pg_dump fetches the list of tables it locks them after this step so they dont disappear to figure out inheritance relationships

  /*
   * Extract data from result and lock dumpable tables.  We do the locking
   * before anything else, to minimize the window wherein a table could
   * disappear under us.
   *
   * Note that we have to save info about all tables here, even when dumping
   * only one, because we don't yet know which tables might be inheritance
   * ancestors of the target table.
   */
result refers to the list of all tables